src/main/java/seedu/dengue/commons/core/GuiSettings.java

```diff
@@ -11,7 +11,7 @@
 public class GuiSettings implements Serializable {
 
     private static final double DEFAULT_HEIGHT = 600;
-    private static final double DEFAULT_WIDTH = 740;
+    private static final double DEFAULT_WIDTH = 800;
```

Comment:
I don't think we're allowed to change this due to feature freeze! We're not allowed to touch any part of the GUI unless it's to fix a genuine bug :O in our case, even if we keep the default width as 740, the user can still handle the truncation by adjusting the window themselves.

Reply:
There's no change to the GUI! The min width is 800 already, and it'll snap to 800 anyways; this just wasn't updated here.

Reply:
Ahh!! Just to be clear: that means that even though we previously set `DEFAULT_WIDTH` to be 740, the GUI was still always 800 instead of 740?

If that's the case then this seems alright :D

Reply:
The GUI was always 800, yup! The minimum height-width was 600-800, so the default width didn't actually do anything haha

Thank you!